src/__tests__/calculations/Dice.test.js

```diff
@@ -0,0 +1,57 @@
+import { Chance } from 'chance';
+
+import {interpolateDie, interpolateD6, modifyMinRequiredRoll} from 'calculations/Dice';
+
+describe('Dice', () => {
+
+  const random = new Chance();
+
+  describe('interpolateDie', () => {
+
+    it('interpolates the % chance to get the mimimum required roll or higher', () => {
+      expect(interpolateDie(3, 2)).toEqual(2/3);
+      expect(interpolateDie(6, 4)).toEqual(0.5);
+      expect(interpolateDie(6, 6)).toEqual(1/6);
+      expect(interpolateDie(10,6)).toEqual(0.5);
```

Comment:
What about 5/6?

Reply:
This test case is handled in the test that runs from line 18-22